### PR TITLE
OSOE-158: Evaluate using the Check Spelling action ⭐

### PR DESCRIPTION
**Pull request by Luko6**
OSOE-158

### CODE REVIEW

File: .github/actions/spelling/excludes.txt [hunk 1]
```diff
@@ -0,0 +1,19 @@
+.jpg$
```
Comment (Piedone) [inline .github/actions/spelling/excludes.txt]: As mentioned under the issues, common configuration should come from the action in the GitHub Actions repo. This and patterns.txt is the same here and there, as well as they seem that they can centrally incorporate what Utility Scripts uses. Perhaps expect.txt too?

File: .github/workflows/check-spelling.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+name: Check Spelling
```
Comment (Piedone) [inline .github/workflows/check-spelling.yml]: The issue requests running this from an existing workflow. Is that not feasible for some reason?

File: Readme.md [hunk 3]
```diff
@@ -100,7 +100,7 @@ When adding a new extension, or significant new features to existing extensions,
...
+- If the feature is user-facing, also add UI test extension method(s) with the [Lombiq UI Testing Toolbox for Orchard Core](https://github.com/Lombiq/UI-Testing-Toolbox/) that assert on some important aspects, and execute them from a new UI test in `Lombiq.OSOCE.Tests.UI` (for inspiration, see the examples there). These methods are also meant to be executed from UI tests in other projects when testing how it integrated with other features. If you've added a demo recipe or sample project to it then utilize that in the test too (see `ExecuteRecipeDirectlyAsync()`). For this, you'll also need to enable the feature in _Lombiq.OSOCE.Tests.recipe_.
```
Comment (Piedone) [inline Readme.md:107]: Did it really not find any issues in submodules?

File: .github/workflows/check-spelling.yml [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+on:
+  pull_request:
+  push:
+    branches:
+      - dev
+
+jobs:
+  check-spelling:
+    name: Check Spelling
+    uses: Lombiq/GitHub-Actions/.github/workflows/check-spelling.yml@issue/OSOE-158
```
Comment (Piedone) [inline .github/workflows/check-spelling.yml]: Test Results now show up under this workflow, despite being connected to the build workflow: 
image

**Review by Piedone — CHANGES_REQUESTED** [review]

File: .github/workflows/check-spelling.yml (hunk 2 above)
Comment (Luko6) [reply .github/workflows/check-spelling.yml]: It is. Somehow I can't read, misunderstood and separated them at the end.

**Review by Piedone — CHANGES_REQUESTED** [review]
See https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/pull/157#discussion_r970990337.

**Review by Piedone — CHANGES_REQUESTED** [review]
Merge, please.

File: Readme.md (hunk 3 above)
Comment (Piedone) [reply Readme.md:107]: Please reply.

Comment (Piedone) [reply Readme.md:107]: @Luko6 This is still unanswered.

**Review by Piedone — CHANGES_REQUESTED** [review]
Again: https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/pull/157#discussion_r970990337

File: Readme.md (hunk 3 above)
Comment (Luko6) [reply Readme.md:107]: It didn't.

Comment (Piedone) [reply Readme.md:107]: And because there aren't any, what I'd find improbable, or it's not working?

**Review by Piedone — APPROVED** [review]

File: .github/workflows/build-and-test.yml [hunk 5]
```diff
@@ -22,3 +22,7 @@ jobs:
       machine-types: "[\"ubuntu-latest\", \"windows-latest\"]"
       build-directory: NuGetTest
       timeout-minutes: 30
+
```
Comment (Piedone) [inline .github/workflows/build-and-test.yml:36]: Something you introduced under https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/pull/220 and affects this too: The Test Results checks are now not under the build but validate PR. See here https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/actions/runs/3289830410/jobs/5421850530 and here: https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/actions/runs/3289830260/jobs/5421850207.

**Review by Piedone — CHANGES_REQUESTED** [review]

**Review by Piedone — APPROVED** [review]

**Review by Piedone — APPROVED** [review]

**Review by Piedone — APPROVED** [review]

**Pull request merged**